
\chapter{Chapter 1. Introduction}%Be sure to include Chapter 1. before you write the name of your chapter. Name all remaining chapters in the same manner.

\section{Background}

The evolution of sexual reproduction is a hallmark of eukaryotic life; various genomic forces led to the evolution of sexual reproduction during the evolutionary trajectory that let to the last eukaryotic common ancestor and its salient features separating it from its prokaryotic ancestors. Once the fundamental features of sexual reproduction were initiated with the last common eukaryotic ancestor, there have been ample pre-zygotic and post-zygotic variations among eukaryotes in terms of their reproductive lifecycles \cite{Goodenough2014-ql}. Within eukaryotic life, embryophytes (land plants) have a reproduction life cycle built around the alternation of generations. Alternations of generations, while being a sexual reproductive lifecycle, works differently than animal reproduction.

In animals, egg and sperm cells form by meiosis, whereas this occurs via mitosis in plants. Within alternation of generations, land plants have an alternation of multicellular diploid and haploid phases. The haploid phase arises from a spore, which in plants is the product of meiosis. Spores can either be the same size (homospory; figure 1.1) or two distinct sizes (heterospory). In heterosporous species, the smaller microspore germinates to form a multicellular male gametophyte, which produces a sperm cell through mitosis; the larger megaspore germinates into a multicellular female gametophyte that produces an egg cell (Figure 1.2). Spores of homosporous species germinate and produce potentially bisexual gametophytes, able to bear both egg and sperm on the same individual. However, it is not uncommon for homosporous plants such as mosses to have separate sexes.

\begin{figure}[ht]
    \centering
    \includegraphics[scale=.7]{Figures/Homosporous_life_cycle.png}
    \caption[The homosporous life cycle in green plants.]{The homosporous life cycle in green plants.}
    \label{fig 1.1}
\end{figure}

\begin{figure}[ht]
    \centering
    \includegraphics[scale=.7]{Figures/Heterosporous_life_cycle.png}
    \caption[The heterosporous life cycle in green plants.]{The heterosporous life cycle in green plants.}
    \label{fig 1.2}
\end{figure}

Extant heterosporous plants consist of three lineages: heterosporous ferns, all seed plants, and heterosporous lycophytes. Most plant species are heterosporous angiosperms (flowering plants), whereas the most common homosporous species are bryophytes, ferns, and lycophytes (the club mosses). All other land plants, including homosporous ferns and lycophytes, are homosporous. There have been at least 11 independent transitions to heterospory from the ancestral condition of homospory in vascular plants, but only three of these transitions are extant (Figure 1.2). The repeated evolution of heterospory represents convergence in vascular plant lineages \cite{Bateman1994-pu}. Heterospory completely negates the possibility of gametophytic self-fertilization and “forces” mitotic (gametophytic) out-crossing in land plants. This out-crossing has been proposed as the selective advantage behind heterospory \cite{Qiu2012-xg}. Modern terrestrial vegetation is dominated by seed plants, and heterospory was an essential prerequisite to evolution of the seed \cite{Petersen2018-wc}. The fundamentally different modes of reproduction makes the transition from homospory to heterospory a non-trivial one; the evolution of heterospory has been labeled as the most significant iterative innovation in the evolution of vascular plants \cite{Bateman1994-pu}.

\begin{figure}[ht]
    \centering
    \includegraphics[width=\textwidth, height=16cm]{Figures/The_lineages_of_green_plants.png}
    \caption[The land plant phylogeny and the multiple origins of heterospory. The purple stars represent the three extant lineages out of the known 11 independent transitions to heterospory in vascular plants. 
    ]{The land plant phylogeny and the multiple origins of heterospory. The purple stars represent the three extant lineages out of the known 11 independent transitions to heterospory in vascular plants. 
    }
    \label{fig 1.3}
\end{figure}


Whereas heterospory has played a crucial role in the evolution of land plants, it is not the only mystery associated with the evolution of heterospory. All the lineages that have seen the independent evolution of heterospory have also coincided with a significant drop in chromosome numbers (figure 1.3). The association between spore type and chromosome number was first reported by Klekowski and Baker \cite{Klekowski1966-zg}, who noted initially that nn average, ferns have n= 57 chromosomes, while the mean angiosperm chromosome number is n = 13.. A more recent meta-analysis of plant chromosome counts \cite{Kinosian2022-uf} substantiated the previous analysis, and the significant differences between heterosporous and homosporous plants remain, with means of 2n = 115 for homosporous plants and 2n = 27.24 for heterosporous plants (Figure 1.4). On average, homosporous plants have 4 times more chromosomes that heterosporous plants, and this difference holds even without the extreme examples of chromosome number, such as the homosporous fern Ophioglossum reticulatum  with 2n = 1440 \cite{Khandelwal1990-nk}, higher than any other known eukaryote.  

\begin{figure}[ht]
    \centering
    \includegraphics[width=\textwidth, height=16cm]{Figures/Chromosome_averages.png}
    \caption[The average number of chromosomes in homosporous and heterosporous plants \cite{Kinosian2022-uf}. The X-axis represents the number of chromosomes , and the Y-axis represents a kernel density representation of frequency.
    ]{The average number of chromosomes in homosporous and heterosporous plants \cite{Kinosian2022-uf}. The X-axis represents the number of chromosomes , and the Y-axis represents a kernel density representation of frequency.
    }
    \label{fig 1.4}
\end{figure}

Earlier approaches to explaining the difference in chromosome numbers between heterosporous and homosporous plants focused on understanding why homosporous plants accumulate chromosomes faster \cite{Haufler2014-ov}. The once-dominant theory was that homosporous plants primarily reproduce via gametophytic selfing, the fusion of gametes produced by mitosis from the same gametophyte (parent). Gametophytic selfing produces completely homozygous zygotes/offspring and would necessitate polyploidy-based redundancy to avoid genetic load, therefore leading to selection for larger genomes \cite{Hickok1978-bw}. However, the tendency of polyploids to act as genetic diploids countered the once prominent gametophytic selfing hypothesis \cite{Haufler1986-qx}. The rejection of the gametophytic selfing-based hypothesis presented by Klekowski \cite{Haufler2014-ov} coincided with a shift from morphology and cytology based exploration to molecular-based studies of plant phylogenetics and evolution.
As phylogenetic research of homosporous plants began to incorporate genomic methods, information from gene copy number patterns indicated that homosporous plants have had lower rates of paleopolyploidy than heterosporous plants despite having more significant chromosome numbers today \cite{Clark2016-de}: \cite{Carta2020-gw}; \cite{Mayrose2021-rw}. Instead, compared to angiosperms, high chromosome numbers in homosporous plants seem to result from higher retention of chromosomes from the fewer rounds of polyploidy \cite{Barker2009-oi}; \cite{Marchant2021-kp}.It seems that heterosporous plants go through higher rates of fractionation, i.e. the potentially permissive loss of duplicate gene and regulatory elements resulting from the relaxed effect of purifying selection on duplicated genomic elements. Higher retention/lower rates of fractionation, rather than increase in chromosome numbers, suggests that homosporous lineages are not outliers that accumulate chromosomes faster than non-homosporous lineages. Instead, it suggests that heterosporous lineages have perhaps undergone higher rates of paleopolyploidy and genome downsizing via reduction in chromosome numbers \cite{Barker2009-oi}; \cite{Clark2016-de}; \cite{Li2021-rk}; \cite{Liu2019-eb}; \cite{Wang2021-du}; \cite{Carins_Murphy2017-bv}.